### PR TITLE
perf(popup.el): enable lexical binding

### PR DESCRIPTION
**Pull request by LemonBreezes**
Upon inspecting the code, I do not see anything can present a problem with enabling `lexical-binding`. One day Emacs will enable lexical binding by default so it's better merge this earlier to reduce the risk of a mass config breakage event.

### CODE REVIEW

**Comment by jcs090218** [comment]
Sounds good to me. Is there a reason this is still draft? 🤔

**Review by jcs090218 — APPROVED** [review]

**Comment by LemonBreezes** [comment]
Oh, I was going to review the writing more but it's ok!

File: popup.el [hunk 1]
```diff
@@ -1,4 +1,4 @@
-;;; popup.el --- Visual Popup User Interface
+;;; popup.el --- Visual Popup User Interface -*- lexical-binding: t -*-
```
Comment (jcs090218) [inline popup.el:1]: The standard is two spaces after the description. 🤔 Can you change it for me? Thanks!

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
> Oh, I was going to review the writing more but it's ok!

No worries. I can wait until this is ready! Just let me know when it's ready to merge! :)

**Comment by LemonBreezes** [comment]
> > Oh, I was going to review the writing more but it's ok!
> 
> No worries. I can wait until this is ready! Just let me know when it's ready to merge! :)

I just force-pushed and it closed the PR. Is it possible to re-open this one?

**Pull request closed without merging**